modules/sc-mesh-secure-deployment/src/nats/comms_nats_controller.py

```diff
@@ -1155,10 +1156,57 @@ def __cbma_process_alive_check(self):
         for process_name in terminated_processes:
             del self.__cbma_processes[process_name]
 
+    def __validate_response(
+        self, response: requests.Response, config: ConfigType, status_type: StatusType
+    ) -> str:
+        """
+        Validate response
+        :param response: HTTP response
+        :param config: config
+        :param status_type: status type
+        """
+        # validation for response parameters
+
+        self.__comms_controller.logger.debug(
+            "validating response: %s, Config %s", response.text.strip(), config
+        )
+        status: str = "FAIL"
```

Comment:
`status_type` doesn't seem to be used in this function, is this just temporary or did you intend to pass it by reference (a workaround is to wrap it in a list) and change its value instead of using this local `status` variable? :thinking:

Reply:
I was first directly updating status from validation, but after removed it added return value.   Need to cleat that out.

Reply:
[34fcb4a](https://github.com/tiiuae/mesh_com/pull/389/commits/34fcb4a106727a4f90a0c8bc7a3291406f16aea1)